api/admin.js

```diff
@@ -20,6 +22,57 @@ router.put("/:email", async (req, res) => {
   return res.json({ hackerProfile: newHackerProfile });
 });
 
+router.get("/profiles", async (req, res) => {
+  const Op = sequelize.Op
+  const { query } = req.query;
+  const flexQuery = '%' + query + '%'
+  try {
+    const profiles = await models.HackerProfile.findAll({
+      where: {
+        [Op.or]: [
+          {
+            email: {
+              [Op.like]: flexQuery
+            }
+          },
+          {
+            firstName: {
+              [Op.like]: flexQuery
+            }
+          },
+          {
+            lastName: {
+              [Op.like]: flexQuery
+            }
+          },
+        ]
+      }
+    });
+    return res.json({
+      profiles
+    });
+  } catch (e) {
+    return res.status(500).json({ error: e.message });
+  }
+});
+
+router.post("/updateRole", async (req, res) => {
+  try {
+    const { email, role } = req.body
+    const result = await models.HackerProfile.update({
+      role: role
+    }, {
+      where: {
+        email: email
+      }
+    })
+
+    return res.json({ result: result })
```

Comment:
Can you make this return `{ success: result }` instead? we decided to make the return object come from the `success` key across all routes

Reply:
will do boss